todo_app/app.py

```diff
@@ -8,8 +8,15 @@
 
 @app.route('/')
 def index():
-    return 'Hello World!'
-
+    items = sessionitems.get_items()
+    return render_template('index.html', items=items)
 
+@app.route('/add_item', methods=['POST'])
+def add():
+    todoTitle = request.form.get('title')
+    sessionitems.add_item(todoTitle)
+    items = sessionitems.get_items()
+    return render_template('index.html', items=items)
```

Comment:
Instead of these two lines, you should return a redirect to the index page.

Benefits include:
- Reduce duplicated code
- The user sees the index URL in their browser
- Refreshing the page won't resubmit the form